### ISSUE TITLE
Recycled Experiment #3 HA & Clay combined

### ISSUE THREAD

**Issue by Carolynwamg** [AguaClara/humic_acid#39]
1. 6L water 
2. Combine known concentrations of clay & HA 
3. Compare results

**Comment by monroews** [comment]
I understand that these experiments are taking longer than we had planned.  The idea was that you finish the calibration and verification that the meters can measure clay and humic acid mixtures in just a few days of testing. These experiments are all very quick because you are just measuring samples that aren't changing with time. Thus each sample should only take a few minutes. I'd like to understand why you are leaving the apparatus running for long periods of time. I've noticed that your experiment is running even when no one is around. Could you post one of your example plots from ProCoDA showing the readings as a function of time when you run one of your tests? From that you can figure out how long a test really needs to take. I'm guessing that it should take no longer than perhaps 5 minutes per test, but you'll have to check the data to know.

You can also work with @HannahSi to plot your data from ProCoDA. THere is no read to write down readings! Let me know if there is any reason why you can't finish up these readings of all the mixtures this coming week!

**Comment by Carolynwamg** [comment]
There were consecutive days where the absorbance readings from the spectrophotometer are constantly higher than 1, even after flushing out the systems and cleaning the cuvettes. Normally it takes half an hour for HA to reach steady state, and clay takes even longer. 
We should be able to finish the mixture readings this week.

**Comment by monroews** [comment]
What method did you find helpful for cleaning the spectrophotometer. 

Could you share a graph showing the change in the readings as a function of time?
What is the hydraulic residence time in the tubing and sample cell?

**Comment by WalterGuardado** [comment]
when we finish doing the experiment, we disconnect the entrance tube then we connect it again in a bottle with 2-3 liter of DI water to pump inside the system and we discard these water for the last tube. (and if is necessary we replace those the dirty tube)

**Comment by WalterGuardado** [comment]
when we did the experiment JUST WITH CLAY, we spend an hour on each step, the total, we spend 3 hour   
conc. clay ----- time----- UNT---------ABSORBANCE
0.16 mg/L -----1 h------  12----------- 0.05
0.6 mg/L------ 1 h------   120--------- 0.227
6 mg/L-------- 1h-------   1000.8 ---- 0.903

**Comment by monroews** [comment]
Great. Can you show a graph from the ProCoDA data file for turbidity and absorbance? If you have questions about how to do that @HannahSi can help.

**Comment by monroews** [comment]
I see that you aren't using ProCoDA to collect data and I believe you may be writing down the numbers from the meters. I think this is an opportunity for the AguaClara team to learn what needs to change in our training process so that students are more comfortable using the laboratory tools that we provide. @MattCimini @JustinConneely @rosiekrasnoff @HannahSi

**Comment by JonathanDavidHarris** [comment]
I'll make sure to go over ProCoDa with them on Thursday. @monroews  Does their spectrophotometer connect like a turbidimeter?

**Comment by monroews** [comment]
Yes! ProCoDA is already collecting the data. I spent a week last fall writing the code so they could collect data with ProCoDA. I recommend using the short experiment method of collecting data with a new file for each dataset so it is easy to find the data in the data files.

**Comment by mlee1721** [comment]
![image](https://user-images.githubusercontent.com/43385922/54152563-e9421400-4413-11e9-85a6-74074c52a4c5.png)

ProCoDa data of absorbance vs. time from our 15 mg/L Humic Acid mixtures.  From the looks of it, experiments should take about 15-20 minutes for a good reading of steady state?

Our trials for 1000 NTU clay exceeded the capabilities for the machine readings (Influent Turbidity meter maxed at 1100 NTU, spectrophotometer maxed at absorbance of 2).

We plan to do 500 NTU to get another round of data since it's hard to learn anything meaningful from 1000 NTU clay trial.

**Comment by monroews** [comment]
Ideally plot the data for a single trial with time starting when the humic acid was added. 
Different initial concentrations of humic acid are then different plots on this same graph with each plot starting at time zero for when the concentration was changed.

Good insight that the experiments can be short.  I can't tell what you are looking at on this graph because nothing is labeled. When did you increase the humic acid concentration? Set that time equal to zero!
You might also have insights about how to speed this up. 

Sounds like you took the concentration of clay a bit too high. 500 NTU seems like a reasonable next step.
Have you plotted and analyzed the data that you've collected thus far? I always recommend analyzing data as you go to make sure that your experimental methods are getting you useful data.
